spec/workers/ons_update_check_worker_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "spec_helper"
+
+UPDATE_RSS = "".freeze
+NO_UPDATE_RSS = "".freeze
+
+RSpec.describe OnsUpdateCheckWorker do
+  describe "#perform" do
+    context "with no applicable update in the RSS feed" do
+      before do
+        stub_request(:get, "https://geoportal.statistics.gov.uk/api/feed/rss/2.0?q=PRD_ONSPD&sort=Date%20Created%7Ccreated%7Cdesc").to_return(status: 200, body: ons_rss(last_updated: Time.zone.now - 1.month), headers: {})
```

Comment:
Don't need these as they're the defaults:

```suggestion
        stub_request(:get, "https://geoportal.statistics.gov.uk/api/feed/rss/2.0?q=PRD_ONSPD&sort=Date%20Created%7Ccreated%7Cdesc").to_return(body: ons_rss(last_updated: Time.zone.now - 1.month))
```

Reply:
Done